docs/ci-cd/index.md

```diff
@@ -35,5 +35,12 @@ Work In Progress
 ## Tools
 
 -   [Daily Fail Summary](tools/daily_fail_summary.md)
+-   Spellcheck
+    -   Currently uses rojopolis/spellcheck-github-actions, a PySpelling-based
```

Comment:
should we maybe link to https://github.com/marketplace/actions/github-spellcheck-action#configuration for documentation ?

Reply:
Should be linked now.